### PR TITLE
Manage & review docs tests

### PR DESCRIPTION
**Pull request by gokul-sol**
**Before creating a pull request make sure that:**

- [ X] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [X ] tests have been updated / new tests has been added (if needed)
- [X ] the `enable_keep_helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-4095
https://tools.hmcts.net/jira/browse/PRL-4096

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by balaganesanhmcts — APPROVED** [review]

**Review by yogendraU — APPROVED** [review]

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -27,13 +27,22 @@ def secrets = [
...
+                secret('courtadmin-swansea-testuser', 'COURTADMIN_SWANSEA_TESTUSER'),
+                secret('judge-testuser-one', 'JUDGE_TESTUSER_ONE'),
+                secret('judge-testpassword', 'JUDGE_TESTPASSWORD')
+        ],
         'ccd-${env}'      : [
                 secret('postcode-info-address-lookup-token', 'ADDRESS_LOOKUP_TOKEN'),
                 secret('ccd-api-gateway-oauth2-client-secret', 'API_GATEWAY_IDAM_SECRET'),
                 secret('ccd-admin-web-oauth2-client-secret', 'ADMIN_WEB_IDAM_SECRET'),
                 secret('idam-data-store-client-secret','IDAM_OAUTH2_DATA_STORE_CLIENT_SECRET'),
-                secret('idam-data-store-system-user-username','IDAM_DATA_STORE_SYSTEM_USER_USERNAME'),
```
Comment (AlokDatta) [inline Jenkinsfile_nightly:35]: Sorry @gokul-sol , dont we need the user name anymore?

File: test/end-to-end/crossbrowser/supportedBrowsers.js [hunk 2]
```diff
@@ -37,15 +37,15 @@ const supportedBrowsers = {
...
-    chrome_win_latest: {
-      browserName: 'chromium',
-      platformName: LATEST_WINDOWS,
-      browserVersion: 'latest-1',
-      'sauce:options': {
-        name: 'PRL: WIN_CHROME_LATEST',
-        screenResolution: '1600x1200'
-      }
-    },
+    // chrome_win_latest: {
```
Comment (yogendraU) [inline test/end-to-end/crossbrowser/supportedBrowsers.js:40]: can you please remove commented lines?

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: Jenkinsfile_nightly (hunk 1 above)
Comment (gokul-sol) [reply Jenkinsfile_nightly:35]: @AlokDatta good catch, added it now. Thanks.

File: test/end-to-end/crossbrowser/supportedBrowsers.js (hunk 2 above)
Comment (gokul-sol) [reply test/end-to-end/crossbrowser/supportedBrowsers.js]: @yogendraU this is now removed

File: charts/prl-ccd-definitions/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: Private Law - CCD Definitions
 name: prl-ccd-definitions
-version: 0.0.18
+version: 0.0.19
```
Comment (AlokDatta) [inline charts/prl-ccd-definitions/Chart.yaml:4]: We do not need this bump

**Review by AlokDatta — CHANGES_REQUESTED** [review]

File: charts/prl-ccd-definitions/Chart.yaml (hunk 3 above)
Comment (gokul-sol) [reply charts/prl-ccd-definitions/Chart.yaml]: @AlokDatta Reverted to 0.0.18, can you please check again.

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**